docs/conf.py

```diff
@@ -0,0 +1,49 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# Warning: do not change the path here. To use autodoc, you need to install the
+# package first.
+
+from typing import List
+
+# -- Project information -----------------------------------------------------
+
+project = "fastjet"
+copyright = "2021, Scikit-HEP"
```

Comment:
```suggestion
copyright = "2021, The Scikit-HEP admins"
```

Reply:
https://github.com/scikit-hep/cookie is generating this by default; if you prefer the shorter version, also update it there.

Reply:
Thanks, have just made the suggestion to Henry.